index.html

```diff
@@ -47,6 +48,157 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
+      </ul>
+      <a href="#" class="tag">
+        <button type="button" class="btn btn-success see-button">See Project</button>
+      </a>
+    </div>
+
+    <div class="card">
+      <div class="card-image car"></div>
+      <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+      <ul class="alist">
+        <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+        <li><a href="#" class="anc card-a2">CSS</a></li>
+        <li><a href="#" class="anc card-a3">Javscript</a></li>
+        <li><a href="#" class="anc card-a4">html</a></li>
+      </ul>
+      <a href="#" class="tag">
+        <button type="button" class="btn btn-success see-button">See Project</button>
+      </a>
+    </div>
+
+    <div class="card">
+      <div class="card-image car"></div>
+      <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+      <ul class="alist">
+        <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+        <li><a href="#" class="anc card-a2">CSS</a></li>
+        <li><a href="#" class="anc card-a3">Javscript</a></li>
+        <li><a href="#" class="anc card-a4">html</a></li>
+      </ul>
+      <a href="#" class="tag">
+        <button type="button" class="btn btn-success see-button">See Project</button>
+      </a>
+    </div>
+
+    <div class="card">
+      <div class="card-image car"></div>
+      <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+      <ul class="alist">
+        <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+        <li><a href="#" class="anc card-a2">CSS</a></li>
+        <li><a href="#" class="anc card-a3">Javscript</a></li>
+        <li><a href="#" class="anc card-a4">html</a></li>
+      </ul>
+      <a href="#" class="tag">
+        <button type="button" class="btn btn-success see-button">See Project</button>
+      </a>
+    </div>
+
+    <div class="card">
+      <div class="card-image car"></div>
+      <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+      <ul class="alist">
+        <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+        <li><a href="#" class="anc card-a2">CSS</a></li>
+        <li><a href="#" class="anc card-a3">Javscript</a></li>
+        <li><a href="#" class="anc card-a4">html</a></li>
+      </ul>
+      <a href="#" class="tag">
+        <button type="button" class="btn btn-success see-button">See Project</button>
+      </a>
+    </div>
+
+    <div class="card">
+      <div class="card-image car"></div>
+      <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+      <ul class="alist">
+        <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+        <li><a href="#" class="anc card-a2">CSS</a></li>
+        <li><a href="#" class="anc card-a3">Javscript</a></li>
+        <li><a href="#" class="anc card-a4">html</a></li>
+      </ul>
+      <a href="#" class="tag">
+        <button type="button" class="btn btn-success see-button">See Project</button>
+      </a>
+    </div>
+
+ <!-- About me -->
```

Comment:
:orange_circle: [OPTIONAL I think it will be better if you wrap everything that related to the about section in a parent section container as follow :arrow_down: 
```
<section id="about-section">
    paste about section content here
</section> 
```